WorkloadManagementSystem/Client/ServerUtils.py

```diff
@@ -37,26 +40,4 @@ def getPilotAgentsDB():
   return getDBOrClient(PilotAgentsDB, serverName)
 
 
-def getTaskQueueDB():
```

Comment:
Why does this go away ?

Reply:
I really don't like this module, because of what I have added in the docstring of it. 
And what I removed from here is what is unused.

Reply:
We have discussed this multiple times. The service handler is as thin layer as possible on top of the corresponding DB class. This makes utilities like this one possible and adding to the overall efficiency. This design choice clearly has advantages.

Reply:
Yes I agree in general, but in this case there's a "WMSAdministratorHandler" on top of PilotAgentsDB, which is also on top of JobDB, and the 2 don't have exactly the same structure. And in this module there was also MatcherHandler treated like TaskQueueDB, and these 2 are quite different, with MatcherHandler adding quite some logic.

It's not that I don't like the pattern. I don't like this specific module, it is very error prone.

Reply:
Yes, I agree that here we have created quite a mess with the WMSAdministrator, may be this should be revised. E.g. make a new service, let say PilotManager to work in conjunction with the PilotXXXDB's and move the corresponding calls from teh WMSAdministrator.

Reply:
Yes I agree that we can create a `PilotAgentsHandler.py` that exposes only calls to `PilotAgentsDB`. But I will not do it in this PR.